src/components/Toaster/README.md

```diff
@@ -16,34 +16,41 @@ import {toaster} from '@yandex-cloud/uikit';
 Toaster is **Singleton**, so when initialized in different parts of the application, the same instance will be returned.
 On initialization it is possible to pass className, that will be assigned to dom-element which is wrapping all toasts.
 
+#### Constructor arguments
+
+| Parameter       | Type      | Default     | Description                                         |
+| :-------------- | :-------- | :---------- | :-------------------------------------------------- |
+| additionalClass | `string`  | `undefined` | Custom class name to add to the component container |
+| mobile          | `boolean` | `false`     | Configuration that manages mobile/desktop views     |
```

Comment:
Unfortunately, we can\`t use mobile context because toaster  is rendering outside the main application